mmv1/third_party/terraform/services/compute/resource_compute_region_target_https_proxy_test.go.erb

```diff
@@ -349,3 +350,290 @@ resource "google_compute_region_ssl_certificate" "foobar2" {
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+			{
+				Config: testAccComputeRegionTargetHttpsProxy_withForwardingRule_withSslPolicy(context),
+			},
+			{
+				ResourceName:      "google_compute_region_target_https_proxy.default-https",
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+func testAccComputeRegionTargetHttpsProxy_withForwardingRule(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+resource "google_compute_forwarding_rule" "default-https" {
+  project               = "%{project_id}"
+  region                = "us-central1"
+  name                  = "https-frwd-rule-%{resource_suffix}"
+  load_balancing_scheme = "INTERNAL_MANAGED"
+  target                = google_compute_region_target_https_proxy.default-https.self_link
+  network               = google_compute_network.ilb_network.name
+  subnetwork            = google_compute_subnetwork.ilb_subnet.name
+  ip_address            = google_compute_address.consumer_address.id
+  ip_protocol           = "TCP"
+  port_range            = "443"
+  allow_global_access   = "true"
+  depends_on            = [google_compute_subnetwork.ilb_subnet2]
+}
+
+resource "google_compute_region_backend_service" "default" {
+  project               = "%{project_id}"
+  region                = "us-central1"
+  name                  = "backend-service-%{resource_suffix}"
+  protocol              = "HTTPS"
+  port_name             = "https-server"
+  load_balancing_scheme = "INTERNAL_MANAGED"
+  session_affinity      = "HTTP_COOKIE"
+  health_checks         = [google_compute_region_health_check.default.self_link]
+  locality_lb_policy    = "RING_HASH"
+
+  # webscoket handling: https://stackoverflow.com/questions/63822612/websocket-connection-being-closed-on-google-compute-engine
+  timeout_sec = 600
+
+  consistent_hash {
+    http_cookie {
+      ttl {
+        # 24hr cookie ttl
+        seconds = 86400
+        nanos   = null
+      }
+      name = "X-CLIENT-SESSION"
+      path = null
+    }
+    http_header_name  = null
+    minimum_ring_size = 1024
+  }
+
+  log_config {
+    enable      = true
+    sample_rate = 1.0
+  }
+}
+
+resource "google_compute_region_health_check" "default" {
+  project             = "%{project_id}"
+  region              = "us-central1"
+  name                = "hc-%{resource_suffix}"
+  timeout_sec         = 5
+  check_interval_sec  = 30
+  healthy_threshold   = 3
+  unhealthy_threshold = 3
+
+  https_health_check {
+    port         = 443
+    request_path = "/health"
+  }
+}
+
+resource "google_compute_region_target_https_proxy" "default-https" {
+  project          = "%{project_id}"
+  region           = "us-central1"
+  name             = "https-proxy-%{resource_suffix}"
+  url_map          = google_compute_region_url_map.default-https.self_link
+  ssl_certificates = [google_compute_region_ssl_certificate.foobar0.self_link]
+
+  lifecycle {
```

Comment:
Why do we need create_before_destroy here?

Reply:
After re-testing, those fields don't appear to be as necessary as I expected, I will be removing them.